docs/guide.rst

```diff
@@ -98,20 +98,32 @@ Here's an example how to use it.
     Signing Declare transactions is possible only with Accounts having `__validate__` entrypoint (with `supported_tx_version = 1`).
 
 
-Deploying new contracts
------------------------
+Deploying contracts
+-------------------
 
-Here's how you can deploy new contracts:
+Simple declare and deploy
+#########################
 
-.. codesnippet:: ../starknet_py/tests/e2e/docs/guide/test_deploying_new_contracts.py
+The simplest way of declaring and deploying contracts on the StarkNet is to use the :ref:`Contract` class.
+
+.. codesnippet:: ../starknet_py/tests/e2e/docs/guide/test_simple_declare_and_deploy.py
     :language: python
     :dedent: 4
 
 
-Deploying new contracts with Universal Deployer Contract (UDC)
---------------------------------------------------------------
+Simple deploy
+#############
+
+If you already know a class_hash of a contract you want to deploy just use the :meth:`Contract.deploy_contract`.
```

Comment:
Let's add info that it will deploy using funds from your account and that UDC is used beneath it.